english/change-change/change-change

```diff
@@ -0,0 +1 @@
+-All of a sudden everything stops, and you find yourself in the middle of a clothing store, where all of the merchandise is made with coins. You look down and realize that your clothes are made of coins as well. You decide it's time to change the change on you, and gear up for your next adventure. Time to change change.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.